atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/ConcludedTransactionsUpdaterTask.java

```diff
@@ -81,6 +84,10 @@ private void schedule() {
 
     @VisibleForTesting
     void runOneIteration() {
+        if (!isInitializedSupplier.getAsBoolean()) {
+            return;
```

Comment:
hmm having to write this init logic everywhere is also pretty jank

Reply:
Ideas I had that came to mind: (1) An optional-esque interface, which will run a lambda if it is initialized, or (2) an abstract class which simply performs this check (should be done in a private method so sub-classes cannot override), but tbh considering this is only in 2(?) places, it should be ok to keep as is.